apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,60 @@
+/*Some necessary adjustments*/
```

Comment:
General css is good, but you overcomplicated based on the basic html that we are using. Try to keep it as simple as possible. I'm sure you don't need this selector ```section:not(:first-child), #ingredients, #directions ``` to target it, for example.
```
ul li, #ingredients li {
  list-style-type: none;
}
```
I have the feeling this rule is targeting the same ```li```s
And the same for the ```ol```